src/interfaces/facades/IBlobsFacade.sol

```diff
@@ -29,4 +31,67 @@ interface IBlobsFacade {
     /// @param size Blob size.
     /// @param bytesReleased Network capacity bytes released.
     event BlobDeleted(address indexed subscriber, bytes32 hash, uint256 size, uint256 bytesReleased);
+
+    /// @dev Get a list of added blobs.
+    /// @param size Maximum number of added blobs to return.
+    /// @return blobs List of added blobs.
+    function getAddedBlobs(uint32 size) external view returns (BlobTuple[] memory blobs);
+
+    /// @dev Get information about a specific blob.
+    /// @param blobHash Blob blake3 hash.
+    /// @return blob Information, including its hash, size, metadata hash, subscribers and status.
+    function getBlob(string memory blobHash) external view returns (Blob memory blob);
+
+    /// @dev Get status of a specific blob for a subscriber.
+    /// @param subscriber The address of the subscriber.
+    /// @param blobHash Blob blake3 hash.
+    /// @param subscriptionId Identifier used to differentiate blob additions for the same subscriber.
+    /// @return status The status of the blob: Pending, Resolved or Failed.
+    function getBlobStatus(address subscriber, string memory blobHash, string memory subscriptionId)
+        external
+        view
+        returns (BlobStatus status);
+
+    /// @dev Get a list of pending blobs.
+    /// @param size Maximum number of pending blobs to return.
+    /// @return blobs List of pending blobs.
+    function getPendingBlobs(uint32 size) external view returns (BlobTuple[] memory blobs);
+
+    /// @dev Get the total count of pending blobs.
+    /// @return Total number of pending blobs.
+    function getPendingBlobsCount() external view returns (uint64);
+
+    /// @dev Get the total size in bytes of pending blobs.
+    /// @return Total size of pending blobs in bytes.
+    function getPendingBytesCount() external view returns (uint64);
+
+    /// @dev Get the storage usage for an account.
+    /// @param addr The address of the account.
+    /// @return usage The storage usage showing total size of all blobs managed by the account.
+    function getStorageUsage(address addr) external view returns (uint256);
```

Comment:
doesn't this come with account info?

Reply:
correct https://github.com/recallnet/contracts/blob/d5c93b34ed0d5e0d071412426464df3b8dfa90a8/src/wrappers/LibBlob.sol#L401

note: the wrappers were originally designed to mimic how the rust CLI structured everything, though. e.g., `recall storage usage`. although, we could choose to remove this logic from the facades, and let the JS SDK handle it.

Reply:
ah, makes sense. i think better to just match the actor methods and let SDKs build their own UX. that would make for a cleaner boundary + more flex for diff tools